modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -100,22 +99,24 @@ const ClassicEditor = ({
 	useEventListener('resize', onResize, true, window);
 
 	return (
-		<div className={cssClass} id={`${name}Container`}>
+		<div id={`${name}Container`}>
 			<label className="control-label" htmlFor={name}>
 				{title}
 			</label>
 			<Editor
 				className="lfr-editable"
 				config={getConfig()}
-				data={contents}
-				key={toolbarSet}
 				onBeforeLoad={(CKEDITOR) => {
 					CKEDITOR.disableAutoInline = true;
 					CKEDITOR.dtd.$removeEmpty.i = 0;
 					CKEDITOR.dtd.$removeEmpty.span = 0;
 
 					CKEDITOR.on('instanceCreated', ({editor}) => {
 						editor.name = name;
+
+						editor.on('instanceReady', () => {
```

Comment:
I remember we had some `processorsLoaded` event in the past... I'm assuming that's properly fired inside `instanceReady`?

Reply:
We are still firing [`customDataProcessorLoaded`](https://github.com/liferay/liferay-portal/blob/579f29bf87652eab0e97183802a6374708e0698c/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_data_processor.js#L961), but we are no longer using it. Its only listener is in ckeditor.jsp. I guess that is an alternative solution for timing when the data can be set.